.circleci/config.yml

```diff
@@ -48,7 +48,47 @@ jobs:
           command: |
             ./project.rb circle-deploy \
               --project all-of-us-workbench-test --creds-file circle-sa-key.json
-
+  api-build-integration-test:
+    docker:
+      - image: allofustest/workbench:buildimage-0.0.10
+    working_directory: ~/workbench
+    environment:
+      JVM_OPTS: -Xmx3200m
+      TERM: dumb
+    steps:
+      - checkout
+      - run:
+          name: Fetch Submodules
+          command: git submodule update --init
+      - restore_cache:
+          keys:
+          - api-cache-{{ checksum "~/workbench/api/build.gradle" }}
+          - api-cache-
+      - run:
+          name: Validate swagger definitions
+          working_directory: ~/workbench/api
+          command: ./project.rb validate-swagger --project-prop verboseTestLogging=yes
+      - run:
+          working_directory: ~/workbench
+          command: ci/activate_creds.sh api/circle-sa-key.json
+      - run:
+          working_directory: ~/workbench
+          command: cp api/circle-sa-key.json public-api
+      - run:
+          working_directory: ~/workbench
+          command: |
+            gsutil cp \
+              gs://all-of-us-workbench-test-credentials/all-of-us-workbench-test-9b5c623a838e.json \
+              api/src/main/webapp/WEB-INF/sa-key.json
+      - run:
+          working_directory: ~/workbench/api
+          command: ./project.rb integration
```

Comment:
I thought that this wouldn't actually attempt to build the server itself, just run integration tests against the test env. If so, shouldn't need anything relating to WEB-INF etc, right?

Reply:
I believe the service account key will be used to access BigQuery, etc... we may need WEB-INF/sa-key.json.

I don't think we need to worry about exploded-api/WEB-INF/lib below.

Reply:
Confirmed we need WEB-INF/sa-key.json, `./project.rb` requires creds when running integration tests. It reads the creds from this location.